olm/templates/workspace-template.yml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
Is this file needed?

Reply:
Yes, it's needed for the CI pipeline callouts (subject of the TODO mentioned above)